manifests/l/Libretro/RetroArch/Nightly/64-bit/1.10.3.0/Libretro.RetroArch.Nightly.64-bit.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Libretro.RetroArch.Nightly.64-bit
+PackageVersion: 1.10.3.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://buildbot.libretro.com/nightly/windows/x86_64/RetroArch-Win64-setup.exe
+  InstallerSha256: AAB8A8BD4A382011C9417DBC363DB98CF2BE49AB8D0EFF103B93978FE805C64A
```

Comment:
Wrong SHA256 sum. You need 7fae3676a37a2b174692c9fc3d17b0c4db284966049e8630cf93a004aa588c60 for it to work.